dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOptionTest.java

```diff
@@ -90,22 +112,40 @@ public void testConfigureAdditionalMavenRepositories_addingSpringRepository()
     Truth.assertThat(paths).isNotEmpty();
   }
 
+  @Test
+  public void testConfigureAdditionalMavenRepositories_notToUseMavenCentral()
+      throws ParseException {
+    CommandLine commandLine =
+        StaticLinkageCheckOption.readCommandLine(
+            new String[] {"-m", "https://repo.spring.io/milestone", "--no-maven-central"});
+    StaticLinkageCheckOption.setRepositories(commandLine);
+
+    CollectRequest collectRequest = new CollectRequest();
+    RepositoryUtility.addRepositoriesToRequest(collectRequest);
+
+    List<RemoteRepository> actualRepositories = collectRequest.getRepositories();
+    Truth.assertThat(actualRepositories).hasSize(1);
+    Truth.assertThat(actualRepositories.get(0).getHost()).isEqualTo("repo.spring.io");
+  }
+
   @Test
   public void testConfigureAdditionalMavenRepositories_invalidRepositoryUrl()
       throws ParseException {
+    assertMavenRepositoryIsInvalid("foobar");
+    assertMavenRepositoryIsInvalid("_http_file__https");
+    assertMavenRepositoryIsInvalid("localhost/abc");
+    assertMavenRepositoryIsInvalid("http://foo^bar");
+  }
 
-    List<String> invalidUrls =
-        ImmutableList.of("foobar", "_http_file__https", "localhost/abc", "http://foo^bar");
-
-    for (String invalidUrl : invalidUrls) {
-      CommandLine commandLine =
-          StaticLinkageCheckOption.readCommandLine(new String[] {"-m", invalidUrl});
-      try {
-        StaticLinkageCheckOption.configureMavenRepositories(commandLine);
-        Assert.fail("URL " + invalidUrl + " should be invalidated");
-      } catch (ParseException ex) {
-        // pass
-      }
+  private static void assertMavenRepositoryIsInvalid(String repositoryUrl)
+      throws ParseException {
+    CommandLine commandLine =
+        StaticLinkageCheckOption.readCommandLine(new String[]{"-m", repositoryUrl});
```

Comment:
If `readCommandLine` took a varargs of String, these tests would be simpler because you wouldn't need all the explicit arrays.

`readCommandLine("-m", repositoryUrl)`.

Reply:
That's nice. Changed the method to take varargs.